air-script/tests/aux_trace/aux_trace.masm

```diff
@@ -1,3 +1,22 @@
+proc.cache_z_exp
+    padw mem_loadw.4294903304 drop drop
+    # => [z_1, z_0, ...]
+    # Exponentiate z trace_len times
+    mem_load.4294903307 neg dup.0 neq.0
```

Comment:
Here, we are loading not `trace_len` but `log2_trace_len`, right? I wonder if it make sense to mention this in a comment.

Reply:
Added a comment for the stack state:

```
    # Exponentiate z trace_len times
    mem_load.4294903307 neg
    # => [count, z_1, z_0, ...] where count = -log2(trace_len)
```